intermediate_html_css/grid/introduction_to_grid.md

```diff
@@ -35,17 +35,17 @@ Or a column like this:
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-For one-dimensional layouts, Flex offers a convenient tool without having to rely on floats or CSS hacks to align your items properly.
+Flex tek boyutlu düzenler için, float'lara veya CSS kurnazlıklarına gerek kalmadan, bize öğelerimizi hizalamamız için kullanışlı bir araç sunar.
 
-For two-dimensional layouts, you learned a little bit about `flex-wrap`, which allows you to take your flex items and wrap them to the next line. This can be done with either a row that wraps to another row, or a column that wraps to another column.
+İki boyutlu düzenler için, flex öğelerinizi bir sonraki satıra wraplamanızı sağlayan `flex-wrap` hakkında biraz bilgi edinmiştiniz. Bu, başka bir satıra wraplayan bir satır ile veya başka bir sütuna wraplayan bir sütun ile yapılabilir.
```

Comment:
```suggestion
İki boyutlu düzenler için, flex öğelerinizi bir sonraki satıra kaydırmanızı sağlayan `flex-wrap` hakkında biraz bilgi edinmiştiniz. Bu, başka bir satıra sarılan bir satır ile veya başka bir sütuna sarılan bir sütun ile yapılabilir.
```

Reply:
wrap reserved bi keyword olduğu için emin olamamıştım nasıl çevirsem, teşekkürler, referans listesine eklense iyi olabilir